app/models/category.rb

```diff
@@ -1,19 +1,18 @@
-class Category < ActiveRecord::Base
+class Category < ApplicationRecord
   extend ActsAsTree::TreeWalker
 
   acts_as_tree :order => 'title'
-  belongs_to :collection
-  has_and_belongs_to_many :articles, -> { order('title').uniq }
-  attr_accessible :collection_id, :title, :gis_enabled
+  belongs_to :collection, optional: true
+  has_and_belongs_to_many :articles, -> { order('title').distinct }
```

Comment:
Do we not need `attr_accessible` any more in Rails 6?

Reply:
@saracarl No, `attr_accessible` is gone and should not be used anymore (the alternative is to use strong parameters).